### PR TITLE
Added Backlog

### PR DESCRIPTION
**Pull request by gillax**

### CODE REVIEW

**Comment by 40504** [comment]
when trying to log in, both Backlog and Typetalk are redirecting to the nulab.com account, already listed.

**Comment by gillax** [comment]
Oh, I see. Nulab is already there.

**Comment by 40504** [comment]
better to let the PR open, the maintainers will decide to add or not

**Comment by phallobst** [comment]
Please rebase your branch to the current master.

**Comment by gillax** [comment]
I've updated it, thank you.

File: _data/developer.yml [hunk 1]
```diff
@@ -60,6 +60,15 @@ websites:
...
     exception: "2FA settings are service-specific."
 
+  - name: Backlog
+    url: https://backlog.com
+    img: backlog.png
+    tfa:
+      - u2f
+      - totp
+      - sms
+    doc: https://support.backlog.com/hc/en-us/community/posts/360029520833
```
Comment (Carlgo11) [inline _data/developer.yml:70]: Old link doesn't work for me. Here's one that does work:
```suggestion
    doc: https://support.nulab.com/hc/en-us/articles/115001025087
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Logo Santa is here with gifts! 🎅 
Try this logo instead:

**Pull request closed without merging**